packages/backend/src/helpers/attachmentsHelper.ts

```diff
@@ -0,0 +1,13 @@
+import multer from 'multer'
```

Comment:
the logic like this should be in: attachment.service.ts
There is no need to create separate file for it